boltdesignsystem/bolt
# DS-177: update academy color and gradient usage with new brand colors

## Jira

https://pegadigitalit.atlassian.net/browse/DS-177

## Summary

Updates Academy demo pages' color and gradient usage with new brand colors.

## Details

1. Purple gradient is deprecated, new one is called Pink.
1. Indigo gradient is deprecated, new one is called Navy.
1. The two deprecated gradients will continue to work, but they will render Pink and Navy visually respectively.
1. Outdated home page and related CSS are removed.
1. Added berry, wine, and violet as options for icon color.

## How to test

Run the branch locally and check all Academy pages, make sure no gradients look broken.

## Release notes

### Deprecated icon color options:

1. Indigo
1. Blue
1. Green
1. White

These are deprecated due to being off brand. Any existing icons set to blue, green, and white will be converted to `auto`. Ones set to indigo will be converted to `navy`.
